components/DataSchema.tsx

```diff
@@ -78,34 +79,35 @@ const DataSchemaTable: React.FunctionComponent<{
         },
     ];
 
+    // Remove Required Column for now.f
```

Comment:
why the f? :)
```suggestion
    // Remove Required Column for now.
```

Reply:
Good catch.  Now fixed:  https://github.com/ncihtan/htan-portal/pull/440/commits/7f9469481f29ab7b4b6f465ab9946a94298eeda6